### PR TITLE
Feat!: Switch default source import from LocalImport to InlineImport

### PR DESCRIPTION
**Pull request by SebastianMorawiec**
# The problem

LocalImport is bad and not very useful for many reasons
https://zapatacomputing.atlassian.net/browse/ORQSDK-859

# This PR's solution

Switch default task source import to InlineImport

# Checklist

_Check that this PR satisfies the following items:_

- [x] Tests have been added for new features/changed behavior (if no new features have been added, check the box).
- [x] The [changelog file](CHANGELOG.md) has been updated with a user-readable description of the changes (if the change isn't visible to the user in any way, check the box).
- [x] The PR's title is prefixed with `<feat/fix/chore/imp[rovement]/int[ernal]/docs>[!]:`
- [x] The PR is linked to a JIRA ticket (if there's no suitable ticket, check the box).

### CODE REVIEW

File: tests/sdk/v2/test_dsl.py [hunk 1]
```diff
@@ -595,35 +567,6 @@ def test_python_imports_deps(python_imports, expected_req, raises):
         assert list(parsed) == expected_req
 
 
-class TestRefToMain:
```
Comment (SebastianMorawiec) [inline tests/sdk/v2/test_dsl.py:598]: Reminder: this test should not be removed after all. Just not valid for InlineImport

Comment (jamesclark-Zapata) [reply tests/sdk/v2/test_dsl.py:598]: Reminding you about the reminder!

**Review by jamesclark-Zapata — APPROVED** [review]
Before merging, can you run through the e2e tests?

**Comment by SebastianMorawiec** [comment]
> 

All tasks in our E2E Ive located are using InlineImport already anyway. So this PR will not change the behaviour

**Pull request merged**